packages/core/cli/src/commands/generate-template.ts

```diff
@@ -0,0 +1,60 @@
+const process = require('process');
+const path = require('path');
+const fs = require('fs');
+const execa = require('execa');
+
+export default async (args) => {
+  if (!args[0]) {
+    console.error('Error: No output folder provided');
+    process.exit(1);
+  }
+
+  const outputPathName = args[0].toLowerCase();
+
+  const vsfTuConfig = `module.exports = {
```

Comment:
can you refactor this file? Eg. decouple generating a template for config from generating template itself